numpy/lib/function_base.py

```diff
@@ -1949,10 +1949,6 @@ def place(arr, mask, vals):
            [44, 55, 44]])
 
     """
-    if not isinstance(arr, np.ndarray):
-        raise TypeError("argument 1 must be numpy.ndarray, "
-                        "not {name}".format(name=type(arr).__name__))
-
```

Comment:
Weird, it seems that `_insert()` is only used here?  Seems fine, could clean it up to have the right function name in the error, but I don't like this function much anyway ;).  We do raise the error, it just shows up as `_insert() ...`.  (would be nice to know this is tested, but not going to sweat it)

Reply:
The error was already good I think, but `_insert` ended up in the stacktrace. I renamed to `_place` to avoid any confusion.

Reply:
Thanks, seems like a reasonable solution.  Note that if you had a fastcall change to `_place`, it got lost somewhere.